auth/build.gradle

```diff
@@ -26,14 +26,20 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+
+    testOptions {
+        unitTests {
+            includeAndroidResources = true
+        }
+    }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'net.openid:appauth:0.7.0'
+    implementation project(path: ':core')
     testImplementation 'org.mockito:mockito-core:2.10.0'
     testImplementation 'junit:junit:4.12'
+    testImplementation "org.robolectric:robolectric:3.6.1"
+    compile 'org.bitbucket.b_c:jose4j:0.6.3'
```

Comment:
compile is deprecated.  replace with implementation and move above the test dependencies.  Also can you include the dependency in the android bom https://github.com/aerogear/aerogear-parent/blob/master/aerogear-android-sdk-bom/pom.xml, passos is working on moving this to use a more managed system.